armory/datasets/preprocessing.py

```diff
@@ -254,14 +235,6 @@ def canonical_variable_image_preprocess(context, batch):
         )
     check_shapes(tuple(batch.shape), context.x_shape)
     assert batch.dtype == context.input_type
-    # assert tf.math.reduce_min(batch) >= context.input_min
-    # assert tf.math.reduce_max(batch) <= context.input_max
-
-    # for x in batch:
-    #     assert x.dtype == context.output_type
-    #     assert tf.math.reduce_min(x) >= context.output_min
-    #     assert tf.math.reduce_max(x) <= context.output_max
-
```

Comment:
Since this is now passed into the dataloader as a function instead of operating on the actual tensor, should this still check the min & max values? TF was complaining about assert statements, but I can look for another way to implement this logic.